plugins/providers/virtualbox/synced_folder.rb

```diff
@@ -52,12 +53,16 @@ def enable(machine, folders, _opts)
             machine.guest.capability(
               :mount_virtualbox_shared_folder,
               os_friendly_id(id), data[:guestpath], data)
+            fstab_folders.push([os_friendly_id(id), data])
```

Comment:
Using the "shovel" operator `<<` is usually more common in Ruby, e.g.:

```suggestion
            fstab_folders << [os_friendly_id(id), data]
```